binary/defaults/binary_controls.defaults

```diff
@@ -49,9 +49,9 @@
 
 
 ! initial_eccentricity
-! ~~~~~~~~~~~~~~~~~~~~
 
 ! Initial eccentricity of the system
+! ~~~~~~~~~~~~~~~~~~~~
```

Comment:
I'm pretty sure Sphinx will emit a warning if the underline isn't as long as the heading.
```suggestion
! ~~~~~~~~~~~~~~~~~~~~
```